Resolutions/XenobladeX_Resolution/rules.txt

```diff
@@ -3,169 +3,230 @@ titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
 name = Resolution
 path = "Xenoblade Chronicles X/Graphics/Resolution"
 description = Changes the resolution of the game. Made by getdls.
-version = 4
+version = 5
 
-
-[Preset]
-name = 1280x720 (Default)
+[Default]
 $width = 1280
 $height = 720
 $gameWidth = 1280
 $gameHeight = 720
-$internalRes = 1.0
-$dither = 0.01
-$scaleShader = 1.0
-$scaleBlur = 0.05
+$internalScale = 1.0
+$shadowScale = 1.0
+$skellViewScale = 1.0
 
-// Quality
 
+# SCREEN RESOLUTION
 [Preset]
-name = 1600x900
-$width = 1600
-$height = 900
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 1.0
-$dither = 0.1
-$scaleShader = (720.0/900.0) # factor for any shader that can't handle fractional scaling
-$scaleBlur = 0.5
+name = 720p (Native)
+category = Resolution
+$width = 1280
+$height = 720
 
 [Preset]
-name = 1920x1080 (HD)
+name = 1080p
+category = Resolution
 $width = 1920
 $height = 1080
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = (720.0/1080.0)
-$dither = 0.15
-$scaleShader = 1.0 #(1440.0/1080.0)
-$scaleBlur = 0.5
 
 [Preset]
-name = 2560x1080 (21:9 HD)
+name = 1440p
+category = Resolution
 $width = 2560
-$height = 1080
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 1.0
-$dither = 0.15
-$scaleShader = (1440.0/1080.0)
-$scaleBlur = 0.5
+$height = 1440
 
 [Preset]
-name = 1920x1200 (10:9 HD)
-$width = 1920
-$height = 1200
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 1.0
-$dither = 0.15
-$scaleShader = (1440.0/1200.0)
-$scaleBlur = 0.5
+name = 2160p (4k)
```

Comment:
We've had complaints before about having terms like 4k and 8k in our packs, so we try to avoid them as much as possible.